test-it/ViaTests/OctalTest.via

```diff
@@ -0,0 +1,107 @@
+// Autogenerated Vireo assembly file
```

Comment:
Sorry, I was refering to this file, OctalTest.via

Reply:
It seems unrelated to the change and I don't see it in the testList.json file.

Reply:
Oh, ok. I'll remove that.